src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerStatsService.java

```diff
@@ -477,28 +477,28 @@ private SummaryStatistic getSummaryStatistic(final PrisonKeyWorkerAggregatedStat
                     .totalNumPrisoners(prisonStats.getTotalNumPrisoners().intValue())
                     .totalNumEligiblePrisoners(prisonStats.getTotalNumEligiblePrisoners().intValue())
                     .numPrisonersAssignedKeyWorker(prisonStats.getNumPrisonersAssignedKeyWorker().intValue())
-                    .percentagePrisonersWithKeyworker(percentagePrisonersWithKeyworker(prisonStats.getNumPrisonersAssignedKeyWorker(), prisonStats.getTotalNumEligiblePrisoners()))
+                    .percentagePrisonersWithKeyworker(percentage(prisonStats.getNumPrisonersAssignedKeyWorker(), prisonStats.getTotalNumEligiblePrisoners()))
                     .numProjectedKeyworkerSessions((int) projectedSessions)
-                    .complianceRate(getComplianceRate(prisonStats.getNumberKeyWorkerSessions(), projectedSessions))
+                    .complianceRate(rate(prisonStats.getNumberKeyWorkerSessions(), projectedSessions))
                     .build();
         }
         return null;
     }
 
-    private BigDecimal percentagePrisonersWithKeyworker(final double numPrisonersAssignedKeyWorker, final double totalNumEligiblePrisoners) {
```

Comment:
Made these methods static as they are really util methods that don't alter state, and it makes testing easier. Can change if people find it ugly.

Reply:
I'd quite like to get rid of all the use of Double and move everything to BigDecimal - but that's going to be a bit of work - I've raised it in a tech dept ticket